detectable/src/test/resources/detectables/functional/swift/noisyOutput.txt

```diff
@@ -1,33 +0,0 @@
-Fetching https://github.com/apple/example-package-fisheryates.git
-Fetching https://github.com/apple/example-package-playingcard.git
-Completed resolution in 1.03s
-Cloning https://github.com/apple/example-package-fisheryates.git
-Resolving https://github.com/apple/example-package-fisheryates.git at 2.0.5
-Cloning https://github.com/apple/example-package-playingcard.git
-Resolving https://github.com/apple/example-package-playingcard.git at 3.0.5
```

Comment:
I think you removed this test case that should ignore these lines.

Reply:
So you think it's worth it to keep this file/SwiftCliParserTest?

Reply:
Just add some of the "noise" back into the new test.